setup/fio-setup-basic

```diff
@@ -179,11 +179,15 @@ parse_numa_mem_policy
 parse_numa_cpu_nodes
 
 no=0
+
 for storage in $storages; do
 	if [ "$raw_disk" = 1 ]; then
 		storage_setup="filename=$storage"
 	else
-		storage_setup="directory=$storage"
+		storage_setup="\
+directory=$storage
+filename=fio_testfile
```

Comment:
do you know what is the default filename if it is not specified before? and this is not related to format logic, thus help split this to a single commit.

BTW: you don't need create a new pr, you can just force push to the pr which requires change.

Reply:
what is the default behavior if you don't specify filename here? Which filename it will use?

Reply:
oh, it will create a file named 'task_0.0.0' with size of specified test size in the provided directory
then running different fio tests will create task_0.1.0 , task_0.2.0 , task_0.3.0 , and so on if directory provided is the same

Reply:
I'm afraid this impact other users if they depend on default behavior to have multiple results. From the patch, multiple runs write to single file fio_testfile, am i right?

If this is specific to your needs, can it be some parameter in job yaml?

Reply:
Yes, multiple runs will write to the single file fio_testfile to fulfill specific requirement
I will add a parameter to determine if fio_testfile needs to be specified or not